### PR TITLE
style: allow footer reference to be more generic regarding the type

### PR DESCRIPTION
**Pull request by warlof**
Use closes instead fixes keyword in order to reference an "issue" as it
can be either related to a bug or a feature request according to the
commit typology.

### CODE REVIEW

File: content/next/index.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 The description is a short description of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
 1. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
 1. A footer MAY be provided one blank line after the body.
-  The footer SHOULD contain additional issue references about the code changes (such as the issues it fixes, e.g.,`Fixes #13`).
+  The footer SHOULD contain additional issue references about the code changes (such as the issues it closes, e.g.,`closes #13`).
```
Comment (stevemao) [inline content/next/index.md]: I don't think those two words make too much difference. Maybe we can keep both.

**Comment by bcoe** [comment]
@stevemao @warlof I was just thinking to myself while I did some editing of the spec that the footer is the most poorly defined part of the specification -- I wonder if we'd do well to simply the description and just say it can contain useful meta information about the pull request -- @stevemao out of curiosity, does the parser only currently handle one footer line, I've seen this on the Node.js project:

```
chore: description

commit body.

Refs: #190
Reviewer: bcoe
Reviewer: bob
```

**Comment by stevemao** [comment]
> does the parser only currently handle one footer line,

No, it can handle multiple lines. EG:

```
BREAKING CHANGE: The `ngMessagesInclude` attribute is now its own directive and that must be placed as a **child** element within the element with the ngMessages directive.

Closes #10036
Closes #9338
```

**Comment by bcoe** [comment]
@warlof mind rebasing, I have no objection to switching to closes; if you think it's the right move.

File: content/next/index.md (hunk 1 above)
Comment (stevemao) [reply content/next/index.md]: I would prefer to change this to 

The footer SHOULD contain additional issue references about the code changes (such as the issues it closes, e.g.,`Closes #13`, `Fixes #24` or `Resolves #42`).

Comment (warlof) [reply content/next/index.md]: Which difference are you making between `fixes` and `resolves` ?

Comment (stevemao) [reply content/next/index.md]: No differences

**Comment by warlof** [comment]
@bcoe sorry for the delay, I just rebase up to your last change.

Basically, it's now only replacing the previous `fix` keyword into a `close` one as your last change sounds to be a little more generic than previously.

**Comment by damianopetrungaro** [comment]
Looks good to me

**Pull request merged**